test/unittests/mongoBackend/mongoUpdateContext_withOnchangeSubscriptions_test.cpp

```diff
@@ -289,8 +289,88 @@ static void prepareDatabase(void) {
   connection->insert(SUBSCRIBECONTEXT_COLL, sub2);
   connection->insert(SUBSCRIBECONTEXT_COLL, sub3);
 
+  /* Given that preparation including csubs, we have to init cache */
+  subscriptionCacheInit(DBPREFIX);
```

Comment:
I think I added this trying to solve the unit test problem 2 weeks ago... However, it seems that are the end you are "mocking" the caché using the prepareSubsCache(). Thus, I understand that the two lines above could be removed.

However, I would have prefer to populate from DB. Which "technical" reason in precluding that, needing to use the prepareSubsCache() ?